CHANGELOG.md

```diff
@@ -1,37 +1,40 @@
 # Changelog
 
-## 6.0.0-rc.1
+## 6.0.0
 
-**Note:** This version is a pre-release and is not recommended for production usage.
+Teleport 6.0 is a major release with new features, functionality, and bug fixes.
 
-This release of Teleport contains a number of improvements and bug fixes.
+The major features are [Database Access](https://goteleport.com/teleport/docs/preview/database-access/), open source role-based access control (RBAC), and exposing a easy to use API.
 
-* Added `tsh env` command: [#5395](https://github.com/gravitational/teleport/pull/5395)
-* OSS RBAC: [#5419](https://github.com/gravitational/teleport/pull/5419)
-* Add MySQL support for database access: [#5453](https://github.com/gravitational/teleport/pull/5453)
-* Implement utmp/wtmp support: [#5491](https://github.com/gravitational/teleport/pull/5491)
-* Apply traits to kube, app and database labels: [#5597](https://github.com/gravitational/teleport/pull/5597)
-* Add `kube_public_addr` config field to `proxy_service`:[#5611](https://github.com/gravitational/teleport/pull/5611)
+Users can review the [6.0 milestone](https://github.com/gravitational/teleport/milestone/33?closed=1) on Github for more details.
 
-## 6.0.0-alpha.2
+## New Features
+
+### Teleport Database Access
 
-**Note:** This version is a pre-release and is not recommended for production usage.
+### OSS RBAC
 
-This release of Teleport contains a number of improvements and bug fixes.
+### Client libraries and API
 
-* Enable proxy protocol support for Teleport Kubernetes proxy service: [#5299](https://github.com/gravitational/teleport/pull/5299).
-* Update `tsh db ls` command to include connection information and add `tsh db config` command: [#5319](https://github.com/gravitational/teleport/pull/5319).
-* Fix authentication failure when logging in via CLI with Access Workflows after removing `.tsh` directory: [#5323](https://github.com/gravitational/teleport/pull/5323).
-* Rename Database Access service `database` role to `db`: [#5359](https://github.com/gravitational/teleport/pull/5359).
-* Fix `tsh login` failure when `--proxy` differs from actual proxy public address: [#5380](https://github.com/gravitational/teleport/pull/5380).
+## Improvements
 
-## 6.0.0-alpha.1
+* Added `utmp`/`wtmp` support for SSH in [#5491](https://github.com/gravitational/teleport/pull/5491).
+* Added the ability to set a Kubernetes specific public address in [#5611](https://github.com/gravitational/teleport/pull/5611).
+* Added Proxy Protocol support to Kubernetes Access in [#5299](https://github.com/gravitational/teleport/pull/5299).
+* Added ACME ([Let's Encrypt](https://letsencrypt.org/)) support to make getting and using TLS certificates easier. [#5177](https://github.com/gravitational/teleport/issues/5177).
```

Comment:
We should add docs/details about how to use this functionality when setting up a new Teleport cluster. Do we have these written @klizhentas?